src/commandline_frame.ts

```diff
@@ -53,7 +53,9 @@ const logger = new Logger("cmdline")
 /** @hidden **/
 const commandline_state = {
     activeCompletions: undefined,
-    clInput: (window.document.getElementById("tridactyl-input") as HTMLInputElement),
+    clInput: window.document.getElementById(
```

Comment:
I'm sure we used to only run `prettier` on lines that had changed. This is annoying (and my fault, IIRC).